### PR TITLE
Creation of path for testing with root certificate

### PR DESCRIPTION
**Pull request by ammaboateng**
I created a path under "Integrate OTA Connect" and placed the pages for testing with a root certificate using device credentials, as I noticed they didn't have a path within the documentation and were a little difficult to locate.

### CODE REVIEW

**Comment by lbonn** [comment]
The errors should be fixed by merging https://github.com/advancedtelematic/aktualizr/pull/1493/, because doxygen had been updated in debian testing.

But then that's a bit annoying that the stable docs branch breaks because the doxygen run depends on an unstable container... We should maybe think about using something more reliable here.

File: docs/ota-client-guide/modules/ROOT/pages/test-root-cert.adoc [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ifdef::env-github[]
+
+[NOTE]
+====
+We recommend that you link:https://docs.ota.here.com/ota-client/latest/{docname}.html[view this article in our documentation portal]. Not all of our articles render correctly in GitHub.
+====
+endif::[]
+
+
+//MC: This is intended for the "test" use case of device credentials provisioning.
```
Comment (patrickvacek) [inline docs/ota-client-guide/modules/ROOT/pages/test-root-cert.adoc]: Is this comment still necessary? I think that was just left over from Merlin.

File: docs/ota-client-guide/modules/ROOT/nav.adoc [hunk 2]
```diff
@@ -62,6 +62,11 @@ ifndef::env-github[:pageroot:]
 ** xref:{pageroot}hsm-provisioning-example.adoc[Generate a device certificate using an HSM]
 ** xref:{pageroot}enable-device-cred-provisioning.adoc[Enable and install device certificates]
 
+* xref:{pageroot}test-root-cert.adoc[Testing with a root certificate]
+** xref:{pageroot}provide-testroot-cert.adoc[Register your test root certificate]
+** xref:{pageroot}generatetest-devicecert.adoc[Generate a test device certificate]
+** xref:{pageroot}enable-device-cred-provtest.adoc[Enable and install test device certificates]
```
Comment (patrickvacek) [inline docs/ota-client-guide/modules/ROOT/nav.adoc]: I didn't realize all these pages were hidden until now!

Comment (ammaboateng) [reply docs/ota-client-guide/modules/ROOT/nav.adoc]: @patrickvacek I realized they were hidden after going through the developer portal doc. I'm not sure if all the information is still up to date but I tried to make these easier to find on the doc portal.

**Comment by patrickvacek** [comment]
FYI I'm in the middle of creating release 2020.1 right now, and I just made the 2020.1-docs branch. It might make sense to move this PR to point to that.

**Comment by tkfu** [comment]
Yeah, this doesn't make sense. It's the exact same content as the "Provision Devices" section. No idea why these files exist in the first place, but let's get rid of them.

**Pull request closed without merging**